java/src/test/java/com/google/appengine/tools/pipeline/DatastoreExtension.java

```diff
@@ -0,0 +1,58 @@
+package com.google.appengine.tools.pipeline;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.testing.LocalDatastoreHelper;
+import lombok.extern.java.Log;
+import org.junit.jupiter.api.extension.*;
+
+/**
+ * Junit5 extension to initialize local datastore emulator for tests
+ * Use it in your tests with {@code @ExtendWith(DatastoreExtension.class)}
+ *
+ * TODO: replace with setup for all the pipelines stuff??
+ */
+@Log
+public class DatastoreExtension implements BeforeAllCallback, AfterAllCallback, BeforeEachCallback {
```

Comment:
analogous to the version @jlorper  created for main codebase, but with `ParameterResolver`